Hardcodet.NotifyIcon.Wpf/Source/NotifyIconWpf/Interop/WindowsMessages.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+// ReSharper disable InconsistentNaming
+#pragma warning disable 1591
+
+namespace Hardcodet.Wpf.TaskbarNotification.Interop
+{
+    public enum WindowsMessages : uint
+    {
+        WM_MOUSEMOVE = 0x0200,
+        WM_LBUTTONDOWN = 0x0201,
+        WM_LBUTTONUP = 0x0202,
+        WM_LBUTTONDBLCLK = 0x0203,
+        WM_RBUTTONDOWN = 0x0204,
+        WM_RBUTTONUP = 0x0205,
+        WM_RBUTTONDBLCLK = 0x0206,
+        WM_MBUTTONDOWN = 0x0207,
+        WM_MBUTTONUP = 0x0208,
+        WM_MBUTTONDBLCLK = 0x0209,
+        WM_USER = 0x0400,
+        WM_USER2 = WM_USER + 2,
```

Comment:
Same goes here - I'd just put in `0x402` etc - after all, you also don't do that for the mouse events.

Reply:
I was surprised to see that the WinForms code you posted followed the exact same pattern. You've definitely spent too much time in Win32 land ;)